### ISSUE TITLE
Non-existent symlinks exist in release zip file

### ISSUE THREAD

**Issue by pcr910303** [keycastr/keycastr#144]
While I was investigating a failing `brew cask` install of keycastr:

```shellsession
$ brew cask install keycastr
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
lerna

==> Satisfying dependencies
==> Downloading https://github.com/keycastr/keycastr/releases/download/v0.9.6/Ke
Already downloaded: /Users/josungbin/Library/Caches/Homebrew/downloads/485be960d3588025ca2c53c6393d956f741220afe6c389f0e446b5770e25ec03--KeyCastr.app.zip
==> Verifying SHA-256 checksum for Cask 'keycastr'.
==> Installing Cask keycastr
==> Purging files for version 0.9.6 of Cask keycastr
Error: Failed to quarantine one or more files within /usr/local/Caskroom/keycastr/0.9.6. Here's the reason:
xattr: No such file: /usr/local/Caskroom/keycastr/0.9.6/KeyCastr.app/Contents/Frameworks/Sparkle.framework/Versions/A/Resources/fr_CA.lproj
xattr: No such file: /usr/local/Caskroom/keycastr/0.9.6/KeyCastr.app/Contents/Frameworks/Sparkle.framework/Versions/A/Resources/fr.lproj/fr.lproj

```

So I unzipped the file manually and tried to investigate what is the non-existent files.

```shellsession
$ ls -la KeyCastr.app/Contents/Frameworks/Sparkle.framework/Resources/
total 152
drwxr-xr-x@ 16 josungbin  staff   512B  4 19  2018 ./
drwxr-xr-x@  5 josungbin  staff   160B  4 19  2018 ../
-rw-r--r--@  1 josungbin  staff   720B  4 19  2018 Info.plist
-rw-r--r--@  1 josungbin  staff   1.0K  4 19  2018 License.txt
-rw-r--r--@  1 josungbin  staff   5.8K  4 19  2018 SUModelTranslation.plist
drwxr-xr-x@  5 josungbin  staff   160B  4 19  2018 SUStatus.nib/
drwxr-xr-x@  6 josungbin  staff   192B  4 19  2018 de.lproj/
drwxr-xr-x@  6 josungbin  staff   192B  4 19  2018 en.lproj/
drwxr-xr-x@  6 josungbin  staff   192B  4 19  2018 es.lproj/
drwxr-xr-x@  7 josungbin  staff   224B  4 19  2018 fr.lproj/
lrwxr-xr-x   1 josungbin  staff    84B 10 16 09:11 fr_CA.lproj@ -> /Users/andym/Development/Build Products/Release/Sparkle.framework/Resources/fr.lproj
drwxr-xr-x@  6 josungbin  staff   192B  4 19  2018 it.lproj/
drwxr-xr-x@  6 josungbin  staff   192B  4 19  2018 nl.lproj/
-rwxr-xr-x@  1 josungbin  staff    58K  4 19  2018 relaunch*
drwxr-xr-x@  6 josungbin  staff   192B  4 19  2018 ru.lproj/
drwxr-xr-x@  6 josungbin  staff   192B  4 19  2018 sv.lproj/

$ ls -la KeyCastr.app/Contents/Frameworks/Sparkle.framework/Resources/fr.lproj/
total 24
drwxr-xr-x@  7 josungbin  staff   224B  4 19  2018 ./
drwxr-xr-x@ 16 josungbin  staff   512B  4 19  2018 ../
drwxr-xr-x@  5 josungbin  staff   160B  4 19  2018 SUAutomaticUpdateAlert.nib/
drwxr-xr-x@  5 josungbin  staff   160B  4 19  2018 SUUpdateAlert.nib/
drwxr-xr-x@  5 josungbin  staff   160B  4 19  2018 SUUpdatePermissionPrompt.nib/
-rw-r--r--@  1 josungbin  staff   8.4K  4 19  2018 Sparkle.strings
lrwxr-xr-x   1 josungbin  staff    84B 10 16 09:11 fr.lproj@ -> /Users/andym/Development/Build Products/Release/Sparkle.framework/Resources/fr.lproj
```

I'm not sure why the wrong symlinks are there (my crude guess is that you're locale is french and macOS treats the developer's locale specially?), but I think they should be removed from the release zip file.

**Comment by akitchen** [comment]
Thanks for reporting. This is the first I’ve heard of this, and this cask has been released for a long time — in fact we’re overdue for an update.

It looks like this is an issue within Sparkle, which is broken and unused anyways. We have a little work to do to remove it.

I don’t think this is related to my locale, I’d bet it’s an issue with a newer version of macOS. What version are you using (and is brew doctor otherwise happy?)

**Comment by pcr910303** [comment]
@akitchen Maybe it's a behavior change of `xattr` on macOS 10.15 (that `xattr` balks on non-existent symlinks), but the issue itself remains, the release build shouldn't contain the symlinks to your development directory; (for example your username is exposed! :-))

**Comment by akitchen** [comment]
hi @pcr910303 I recently converted Sparkle to be a cross-project dependency instead of checked-in framework, in order to build for Apple Silicon. I ran your same command on the newer built version of Sparkle and it no longer symlinks out to a path on `andym`'s filesystem.

```
drwxr-xr-x  41 akitchen  staff   1312 Dec 13 16:19 .
drwxr-xr-x   5 akitchen  staff    160 Dec 13 16:19 ..
drwxr-xr-x   3 akitchen  staff     96 Dec 13 16:19 Autoupdate.app
-rw-r--r--   1 akitchen  staff     98 Dec 13 16:19 DarkAqua.css
-rw-r--r--   1 akitchen  staff   1311 Dec 13 16:19 Info.plist
-rw-r--r--   1 akitchen  staff  11859 Dec 13 16:19 SUModelTranslation.plist
-rw-r--r--   1 akitchen  staff  12281 Dec 13 16:19 SUStatus.nib
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 ar.lproj
drwxr-xr-x   3 akitchen  staff     96 Dec 13 16:19 ca.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 cs.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 da.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 de.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 el.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 en.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 es.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 fi.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 fr.lproj
lrwxr-xr-x   1 akitchen  staff      8 Dec 13 16:19 fr_CA.lproj -> fr.lproj
drwxr-xr-x   3 akitchen  staff     96 Dec 13 16:19 he.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 hr.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 hu.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 is.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 it.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 ja.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 ko.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 nb.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 nl.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 pl.lproj
lrwxr-xr-x   1 akitchen  staff     11 Dec 13 16:19 pt.lproj -> pt_BR.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 pt_BR.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 pt_PT.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 ro.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 ru.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 sk.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 sl.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 sv.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 th.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 tr.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 uk.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 zh_CN.lproj
drwxr-xr-x   6 akitchen  staff    192 Dec 13 16:19 zh_TW.lproj
```

The issue you mentioned and the effect on brew cask will be fixed indirectly in v0.9.9. Thank you for opening the issue!
